bin/start.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+SSH_SECRETS_DIR=${SSH_SECRETS_DIR:-/etc/feedhenry/gitlab-shell}
+AUTHORIZED_KEYS_FILE=${AUTHORIZED_KEYS_FILE:-/home/git/.ssh/authorized_keys} 
+
+echo "Initializing authorized_keys file"
+cat $SSH_SECRETS_DIR/repoadmin-id-rsa-pub > $AUTHORIZED_KEYS_FILE
```

Comment:
For cheffed clusters, we only add the keys if they aren't already there https://github.com/fheng/fhcap/blob/master/wrapper-cookbooks/gitlabshell_feedhenry/recipes/default.rb#L35-L63